src/main/java/by/custom/utilcalculator/controller/MessageRouter.java

```diff
@@ -35,36 +35,49 @@ public SendMessage route(final Update update) {
 
     //we receive message from user, check it and handle it if it is ok here
     private SendMessage getCheckInputMessageAndGetAnswer(final Message message) {
-        String usersMessage = message.getText();
-        String chatID = message.getChatId().toString();
-        String answer;
+        final String usersMessage = message.getText();
+        final String chatID = message.getChatId().toString();
+        final String answer;
         try {
             answer = route(usersMessage, chatID);
-        } catch (UtilsborException e) {
+        } catch (final UtilsborException e) {
             e.printStackTrace();
             return new SendMessage(chatID, getExceptionText(e));
         }
         return new SendMessage(chatID, answer);
     }
 
     private String route(final String usersMessage, final String chatID) throws UtilsborException {
-        String answer;
-        switch (usersMessage) {
-            case Command.START -> {
+        final String answer;
+        Command requestingCommand = Command.getCommandByKey(usersMessage);
+        switch (requestingCommand) {
+            case START -> {
                 userProgressManager.createNewUserProgress(chatID);
                 answer = getGreetingMessage();
             }
-            case Command.EAES, Command.OTHER_COUNTRIES ->
-                    answer = userProgressManager.processCarOrigin(usersMessage, chatID);
-            case Command.PHYSICAL_PERSON, Command.JURIDICAL_PERSON ->
-                    answer = userProgressManager.processOwnerType(usersMessage, chatID);
-            case Command.LESS_3_YEARS_AGE, Command.BETWEEN_3_AND_7_YEARS_AGE ->
-                    answer = userProgressManager.processCarAge(usersMessage, chatID);
-            case Command.GASOLINE_TYPE_ENGINE, Command.ELECTRIC_TYPE_ENGINE ->
-                    answer = userProgressManager.processEngineType(usersMessage, chatID);
-            case Command.VOLUME_LESS_1000_CM, Command.VOLUME_BETWEEN_1000_2000_CM, Command.VOLUME_BETWEEN_2000_3000_CM,
-                    Command.VOLUME_BETWEEN_3000_3500_CM, Command.VOLUME_MORE_3500_CM ->
-                    answer = userProgressManager.processEngineVolume(usersMessage, chatID);
+
+            case M1 -> answer = userProgressManager.processTransportType(requestingCommand, chatID);
+
+            case EAES, OTHER_COUNTRIES -> answer = userProgressManager.processCarOrigin(requestingCommand, chatID);
+
+
+            case PHYSICAL_PERSON, JURIDICAL_PERSON ->
+                    answer = userProgressManager.processOwnerType(requestingCommand, chatID);
+
```

Comment:
too many empty lines to my taste. It's not call to action

Reply:
Agree, the code had been cleaned